langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -66,6 +90,11 @@ public isolated function getCodePoint(string str, int index) returns int = @java
 
 # Returns a substring of a string.
 #
+# ```ballerina
+# string intro = "Hello, my name is John";
+# string name = intro.substring(18, 22);
```

Comment:
We need a another example with default value for `endIndex`

Reply:
And maybe that example can use variables for arguments too.